README.md

```diff
@@ -0,0 +1,78 @@
+[![N|Solid](https://renew-wireless.org/figs/cropped-v4-qgkw8d-1024x320.png)](https://renew-wireless.org/)
+
+
+# RENEWLab
+
+
+# Description
+The *RENEWLab* software suite functions as the application layer of the RENEW massive MIMO platform [(official documentation)](https://docs.renew-wireless.org). It provides a user interface through a set of APIs. Users can generate, manipulate, transmit, and receive RF signals on the RENEW hardware by calling these APIs. 
+
+> The design goal of RENEWLab is to provide the RENEW software library and a quick starting point for users to design, program, and run their experiments on the RENEW massive MIMO platform. 
+
+
+# Components
+The RENEWLab software suite consists of four components. 
+
+  1. **Python Development Suite**: 
+     It provides a Python-based library which allows users to rapidly program and test the physical layer and the radio layer in real time. It also provides tools for offline data procesing. Please refer to the [RENEW offical documentation](https://docs.renew-wireless.org/dev-suite/design-flows/python-design-flow/). 
+
+  2. **MATLAB Development Suite**: 
+     It provides a MATLAB-based library which allows users to rapidly develop physical layer algorithms using the MATLAB toolboxes with a highly simplified interface and to perform OTA tests. Please refer to the [RENEW official documentation](https://docs.renew-wireless.org/dev-suite/design-flows/matlab-design-flow/). 
+
+  3. **C++ Development Suite**: 
+     It provides a set of projects for real-time transmission and receiption of RF signals and for a highly parallelized system with large processing throughput. It supports two projects. Please refer to the [RENEW official documentation](https://docs.renew-wireless.org/dev-suite/design-flows/cpp/).
+     - The Sounder framework for running channel sounding experiments
+     - The Beamformer framework (TODO) for running real-time uplink and downlink beamforming
```

Comment:
We can remove Beamformer here. Beamformer is Agora and it will remain a seperate repo. We can update the documentation accordingly. The C++ Development suite is essentially the Sounder.